### PR TITLE
Add initial happy path tests

### LINKED ISSUE #194: Add happy path tests for gcputils
### TL;DR

We are missing some test coverage. Add tests for PROJECT_ID and GOOGLE_PROJECT.

### Detailed design

_No response_

### Alternatives considered

_No response_

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by shan4993**
resolves #194

### CODE REVIEW

File: gcputil/gcputil_test.go [hunk 1]
```diff
@@ -0,0 +1,61 @@
+// Copyright 2023 The Authors (see AUTHORS file)
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
```
Comment (yolocs) [inline gcputil/gcputil_test.go:8]: Fix the lint please

File: gcputil/gcputil_test.go [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+    want string
+  }{
+    {
+      name: "success_project_id",
+      env: map[string]string{
+        "PROJECT_ID":           "projectID",
+        "GOOGLE_PROJECT":       "googleProject",
+        "GOOGLE_CLOUD_PROJECT": "googleCloudProject",
+      },
+      want: "projectIDD", // fix typo
```
Comment (yolocs) [inline gcputil/gcputil_test.go:35]: Here is typo, should be `projectID`

File: gcputil/gcputil_test.go [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+    },
+    {
+      name: "success_google_project",
+      env: map[string]string{
+        "GOOGLE_PROJECT":       "googleProject",
+        "GOOGLE_CLOUD_PROJECT": "googleCloudProject",
```
Comment (yolocs) [inline gcputil/gcputil_test.go:41]: Not blocking but consider also supporting env var `GOOGLE_CLOUD_PROJECT`

**Review by yolocs — CHANGES_REQUESTED** [review]
looking good some nits

File: gcputil/gcputil_test.go (hunk 1 above)
Comment (shan4993) [reply gcputil/gcputil_test.go:8]: ack

File: gcputil/gcputil_test.go (hunk 3 above)
Comment (shan4993) [reply gcputil/gcputil_test.go]: creating an issue and lets circle back #197

File: gcputil/gcputil_test.go (hunk 2 above)
Comment (shan4993) [reply gcputil/gcputil_test.go]: ack

**Review by yolocs — APPROVED** [review]

**Pull request closed without merging**